test/index.js

```diff
@@ -113,6 +113,22 @@ const exportTest = (methodName, genArgs, testFn, extraName = '') =>
 const exportBenchmark = (methodName, genArgs, useTryCatch = false, extraName = '') =>
   exports[`${methodName}${extraName}Benchmark`] = benchmarkGen(methodName, genArgs, useTryCatch, extraName)
 
+exports['h3IsValid_array'] = test => {
+    test.ok(h3node.h3IsValid([0x3fffffff, 0x8528347]), 'Integer H3 index is considered an index');
+    test.ok(
+        !h3node.h3IsValid([0x73fffffff, 0xff2834]),
+        'Integer with incorrect bits is not considered an index'
+    );
+    // TODO: This differs from h3-js, in these cases h3-js would return false rather than throwing
+    test.throws(() => h3node.h3IsValid([]), 'Empty array is not valid');
```

Comment:
This seems like incorrect behavior. Is there a way to wrap in a try/catch? Are we getting these errors from your new code?

Reply:
Yes, although these cases will all throw in h3-node as it is. We would need to change the behavior of h3-node to match h3-js here. Further, these cases will throw in other functions, too, unlike in h3-js where `[0, 0]` is used.

Reply:
I guess this is up to @dfellis, as it's an API question, but my assumption is that `h3IsValid` should never throw for invalid input, since it's the job of the function to recognize it. We could check for valid C input in JS and return `false` before sending to C and getting an error.